doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
+
+## Database
+
+Select the database resource for your setup.
+
+## Redis
+
+Configure the Redis connections details of your Icinga host.  
+If you are running a master-master setup, the first 2 fields are for your configuration master.
```

Comment:
- Which fields are you referring to? (missing screenshot)
- What exactly is a "master-master" setup? This terminology doesn't exist in the Icinga 2 distributed monitoring docs. Better use something like "a high availability zone with two masters".